MAINTAINERS.md

```diff
@@ -17,26 +12,3 @@
 | Peter Nied       | [peternied](https://github.com/peternied)             | Amazon      |
 | Craig Perkins    | [cwperks](https://github.com/cwperks)                 | Amazon      |
 | Ryan Liang       | [RyanL1997](https://github.com/RyanL1997)             | Amazon      |
-
-### Updating Practices
```

Comment:
The below sections are additional maintainers processes and responsibilities.  I'd like to keep them in this document.

Reply:
I'll put these back.